### PR TITLE
Oded/fixed calls rington

### PR DESCRIPTION
**Pull request by odeds**

### CODE REVIEW

File: dist/calls/calls.js [hunk 1]
```diff
@@ -203,12 +203,16 @@
...
-        ["$scope", "CallsSrv", "CallsUiSrv", "CallsStatusEnum", "$log", "CallsErrorSrv", "$timeout", function ($scope, CallsSrv, CallsUiSrv, CallsStatusEnum, $log, CallsErrorSrv, $timeout) {
+        ["$scope", "CallsSrv", "CallsUiSrv", "CallsStatusEnum", "$log", "CallsErrorSrv", "$timeout", "$window", function ($scope, CallsSrv, CallsUiSrv, CallsStatusEnum, $log, CallsErrorSrv, $timeout, $window) {
             'ngInject';
 
             var self = this;
             var callsData = self.scope.callsData;
 
+            var mySound;
+
+            var soundSrc = 'https://dfz02hjbsqn5e.cloudfront.net/general/incomingCall.mp3';
```
Comment (assafshp) [inline dist/calls/calls.js]: use ENV.mediaEndpoint

File: dist/calls/calls.js [hunk 2]
```diff
@@ -249,6 +253,27 @@
                 _fillLoader(bool, methodName);
             }
 
+            function playAudio() {
+                try {
+                        mySound = new $window.Audio(soundSrc);
+                        mySound.addEventListener('ended', function() {
```
Comment (assafshp) [inline dist/calls/calls.js]: no need to check if Audio is supported ? I know that not all browsers supports. Maybe check and $log.error if an error

File: dist/calls/calls.js [hunk 3]
```diff
@@ -249,6 +253,27 @@
...
+                            this.currentTime = 0;
+                            this.play();
+                        }, false);
+                        mySound.play();
```
Comment (assafshp) [inline dist/calls/calls.js]: ended event is for loop ? why ? there is no loop property ?

File: dist/calls/calls.js (hunk 1 above)
Comment (odeds) [inline dist/calls/calls.js]: done

File: dist/calls/calls.js (hunk 2 above)
Comment (odeds) [inline dist/calls/calls.js]: in mdn all brower support even ie, added check anyway

File: dist/calls/calls.js (hunk 3 above)
Comment (odeds) [inline dist/calls/calls.js]: no, just un audio tag, that's the recommend approach for loop

File: dist/calls/calls.js [hunk 4]
```diff
@@ -249,6 +253,31 @@
...
+                    } catch(e) {
+                        $log.error('IncomingCallModalCtrl playAudio failed!' +' err: ' + e);
+                    }
+                }
```
Comment (assafshp) [inline dist/calls/calls.js]: mayber here also write log

Comment (odeds) [inline dist/calls/calls.js]: added log

**Pull request merged**